auslib/web/public/base.py

```diff
@@ -20,7 +21,12 @@
 }
 
 connexion_app = connexion.App(__name__, specification_dir='.', validator_map=validator_map)
-connexion_app.add_api('api.yml', validate_responses=True, strict_validation=True)
+
+import os
+f = os.path.join(os.path.dirname(__file__), 'api.yml')
+stream = file(f, 'r')
+spec = yaml.load(stream)
+connexion_app.add_api(spec, validate_responses=True, strict_validation=True)
```

Comment:
I assume this is just to verify that a dict works for now, and this will change to SpecBuilder later?